package.json

```diff
@@ -131,8 +131,14 @@
     "test:e2e:components-react/remix": "yarn --cwd ./packages/components-react/projects/remix test:e2e",
     "test:e2e:components-vue": "yarn --cwd ./packages/components-vue test:e2e",
     "test:e2e:crawler": "yarn --cwd ./packages/crawler test:e2e",
-    "test:e2e:storefront": "yarn --cwd ./packages/storefront test:e2e",
+    "test:e2e:playwright:storefront": "yarn --cwd ./packages/storefront test:e2e",
+    "test:e2e:playwright:storefront:1/4": "yarn --cwd ./packages/storefront test:e2e:1/4",
+    "test:e2e:playwright:storefront:2/4": "yarn --cwd ./packages/storefront test:e2e:2/4",
+    "test:e2e:playwright:storefront:3/4": "yarn --cwd ./packages/storefront test:e2e:3/4",
+    "test:e2e:playwright:storefront:4/4": "yarn --cwd ./packages/storefront test:e2e:4/4",
```

Comment:
Can be renamed to test:e2e:storefront since we don't have puppeteer anymore.

Reply:
Well, yeah. Anyway, I'd suggest to rename all this kind of stuff after we've migrated really everything within the Monorepo to Playwright.